### PR TITLE
Fix testing class namespaces

### PR DESCRIPTION
**Pull request by peter279k**
# Changed log

- Fixing testing class namespaces to be compatible `PSR-4` autoloader.

### CODE REVIEW

File: tests/unit/TestCase.php [hunk 1]
```diff
@@ -6,7 +6,6 @@
 use function GuzzleHttp\Psr7\parse_response;
 use GuzzleHttp\ClientInterface;
 use GuzzleHttp\Psr7\Response;
-use Prophecy\Argument;
```
Comment (peter279k) [inline tests/unit/TestCase.php:9]: It's declared, but not used. It should be removed.

**Comment by haphan** [comment]
@peter279k  thanks for this pr. Does it break anything or more of inconsistency issue?

**Comment by peter279k** [comment]
This will not break anything and it's only for testing when PHPUnit does unit tests and load namespaces autoloading.

**Comment by haphan** [comment]
I mean before this fix, what was not working?

**Comment by peter279k** [comment]
When using the [refactor](https://github.com/rectorphp/rector), it will present some test classes are not found.

And I fix them to be compatible with PSR-4 autoloader.

**Comment by haphan** [comment]
Let me do a quick review later this week

**Comment by k0ka** [comment]
After 3 years of quick review we were able to merge it =).
https://github.com/php-opencloud/openstack/pull/383
Thank you for your PR.

**Pull request closed without merging**